BlazarBase/src/main/java/com/hubspot/blazar/base/BuildConfig.java

```diff
@@ -9,6 +9,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Objects;
 import com.google.common.base.Optional;
+import com.hubspot.mesos.Resources;
```

Comment:
Let's don't leak model objects from other projects (especially when they are not considered libs) into blazar config.